api/API_VERSIONING.md

```diff
@@ -84,52 +100,43 @@ from clients (Envoy, gRPC) and a large number of control planes, ranging from si
   weighing this input with the first two considerations above.
 
-Following the release of a new major version, the API lifecycle follows a deprecation clock.
-Envoy will support at most three major versions of any API package at all times:
-* The current stable major version, e.g. v3.
-* The previous stable major version, e.g. v2. This is needed to ensure that we provide at least 1
-  year for a supported major version to sunset. By supporting two stable major versions
-  simultaneously, this makes it easier to coordinate control plane and Envoy
-  rollouts as well. This previous stable major version will be supported for exactly 1
-  year after the introduction of the new current stable major version, after which it will be
-  removed from the Envoy implementation.
-* Optionally, the next experimental alpha major version, e.g. v4alpha. This is a release candidate
-  for the next stable major version. This is only generated when the current stable major version
-  requires a breaking change at the next cycle, e.g. a deprecation or field rename. This release
-  candidate is mechanically generated via the
-  [protoxform](https://github.com/envoyproxy/envoy/tree/main/tools/protoxform) tool from the
-  current stable major version, making use of annotations such as `deprecated = true`. This is not a
-  human editable artifact.
-
-An example of how this might play out is that at the end of December in 2020, if a v4 major version
-is justified, we might freeze
-`envoy.config.bootstrap.v4alpha` and this package would then become the current stable major version
-`envoy.config.bootstrap.v4`. The `envoy.config.bootstrap.v3` package will become the previous stable
-major version and support for `envoy.config.bootstrap.v2` will be dropped from the Envoy
-implementation. Note that some transitively referenced package, e.g.
-`envoy.config.filter.network.foo.v2` may remain at version 2 during this release, if no changes were
-made to the referenced package. If no major version is justified at this point, the decision to cut
-v4 might occur at some point in 2021 or beyond, however v2 support will still be removed at the end
-of 2020.
-
-The implication of this API lifecycle and clock is that any deprecated feature in the Envoy API will
-retain implementation support for at least 1-2 years.
-
-We are currently working on a strategy to introduce minor versions
-(https://github.com/envoyproxy/envoy/issues/8416). This will bump the xDS API minor version on every
-deprecation and field introduction/modification. This will provide an opportunity for the control
-plane to condition on client and major/minor API version support. Currently under discussion, but
-not finalized will be the sunsetting of Envoy client support for deprecated features after a year
-of support within a major version. Please post to https://github.com/envoyproxy/envoy/issues/8416
-any thoughts around this.
+A new API minor version is expected to be released once a year, around Q1. This
```

Comment:
This is going to be "at the Q2 release" as things stand.

Reply:
Done.